AirCasting/Utils/Strings.swift

```diff
@@ -45,4 +45,47 @@ struct Strings {
         static let title: String = "Your AirBeam is gathering data."
         static let description: String = "Your AirBeam is gathering data."
     }
+    
+    struct OnboardingGetStarted {
+        static let description: String = "record and map measurements from health and environmental monitoring devices"
+        static let getStarted: String = "Get started"
+    }
+    
+    struct OnboardingNearAir {
+        static let title: String = "How’s the air \nnear you?"
+        static let description: String = "Find and follow a fixed air quality monitor near you and know how clean or polluted your air is right now."
+        static let continueButton: String = "How’s the air \nnear you?"
+    }
+    
+    struct OnboardingAirBeam {
+        static let title: String = "Measure and map \nyour exposure \nto air pollution"
+        static let description: String = "Connect AirBeam to measure air quality humidity, and temperature."
+        static let continueButton: String = "Continue"
+        static let sheetButton: String = "Learn More"
+    }
+    
+    struct OnboardingAirBeamSheet {
+        static let sheetTitle: String = "How AirBeam works?"
+        static let sheetDescription_1: String = "Your AirBeam is gathering data."
+        static let sheetDescription_2: String = " mobile "
+        static let sheetDescription_3: String = "mode, the AirBeam captures personal exposures.\n\n\nIn"
+        static let sheetDescription_4: String = " fixed "
+        static let sheetDescription_5: String = "mode, it can be installed indoors or outdoors to keep tabs on pollution levels in your home, office, backyard, or neighborhood 24/7."
```

Comment:
I feel like this could've been done a bit differently, but let's go with that ;)